lib/main.dart

```diff
@@ -258,8 +260,9 @@ class MyHomePage extends StatelessWidget {
                 final taskTitle = titleController.text.trim();
                 final taskDescription = descriptionController.text.trim();
                 final taskDueDate = dueDateController.text.trim();
-
+                //Majd
```

Comment:
Ensure to handle possible invalid inputs for task title, description, and date to avoid potential runtime errors.